dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -195,6 +200,22 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return *this;
     }
 
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::AddTextBox(hstring id)
+    {
+        ThrowIfMaxInputItemsExceeded();
```

Comment:
Should we check argument to see if it is empty and throw?

Reply:
Me thinks yes :) We havent done any argument validation for string types at all in the builder. We probably should.

Reply:
done